### ISSUE TITLE
Move copy and past extensions into stdmenus

### ISSUE THREAD

**Issue by zapbot** [zaproxy/zaproxy#937]
```
Bit over the top for these to be in their own extensions, and currently they share the
same order which causes an error to be logged.
Minor tidy up.
```

Original issue reported on code.google.com by `psiinon` on 2013-12-05 15:43:05

**Comment by kingthorin** [comment]
Is this still an issue?

If so can you provide more info.

**Comment by thc202** [comment]
It was fixed in 6385805345ebd37f20d7c864a87a954fd329673d, included in 2.4.0.
